model.js

```diff
@@ -9,16 +9,28 @@ function returnTopics(){
     })
 }
 
-
+function returnArticleComments(articleId){
+    return db.query("SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at ASC" ,[articleId])
+    .then(({rows})=>{
+        if(rows.length === 0){
```

Comment:
This logic will wrongly reject for articles with no comments, you should use a checkExists before this, to differentiate between existing articles with no comments (returns []) and non existing articles with no comments (also returns []). Drop me a Dm for any clarification on this :)